### PR TITLE
Fix a few things

### PR DESCRIPTION
**Pull request by dy3l**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -119,14 +119,14 @@
...
-- [KHALID KATKOUT ](https://www.youtube.com/user/khalidnobleman) - Networking - CCNA - GNS3 - IPv6 ...
+- [MaDrasaTech](https://www.youtube.com/user/JasonMoroccan) - Networking - Network Automation Python - Windows Server ...
+- [KHALID KATKOUT](https://www.youtube.com/user/khalidnobleman) - Networking - CCNA - GNS3 - IPv6 ...
 
 ---
 
 ### Open Source Projects
 
-#### JAVASCRIPT
+#### JavaScript
```
Comment (JefferyHus) [inline readme.md:129]: What would be the reason behind changing the titles?

File: readme.md [hunk 2]
```diff
@@ -165,7 +165,7 @@
 - [JSS](https://github.com/mboussaid/JSS) - A minimal `javascript` ibrary that allows you to change the style of elements By [@mboussaid](https://github.com/mboussaid)
 - [VerifyJS](https://github.com/mboussaid/VerifyJS) - A minimal `javascript` ibrary that allows to add verification to your inputs By [@mboussaid](https://github.com/mboussaid)
 
-#### JAVA
+#### Java
```
Comment (JefferyHus) [inline readme.md:168]: Same thing here.

File: readme.md [hunk 3]
```diff
@@ -202,7 +202,7 @@
 - [My-Business](https://github.com/GDGSNF/My-Business) - A Business Management Tool by [@yezz123](https://github.com/yezz123)
 - [Athena](https://github.com/GDGSNF/Athena) - web application developed in Python / Flask / SQLite that has two faces. [@yezz123](https://github.com/yezz123)
 
-#### Go/Golang
+#### Go
```
Comment (JefferyHus) [inline readme.md:205]: Here as well.

File: readme.md [hunk 4]
```diff
@@ -217,7 +217,7 @@
 - [Tailwindcss Twitter Page](https://github.com/simoebenhida/tailwindcss-twitter-page) - A twitter profile made with tailwindcss. By [@simoebenhida](https://github.com/simoebenhida)
 - [Tailwindcss Paypal Page](https://github.com/simoebenhida/tailwindcss-paypal-page) - A paypal page made with tailwindcss. By [@simoebenhida](https://github.com/simoebenhida)
 
-#### FLUTTER
+#### Flutter
```
Comment (JefferyHus) [inline readme.md:220]: And also this one.

**Review by JefferyHus — CHANGES_REQUESTED** [review]

**Comment by dy3l** [comment]
Thank you @jEFFERYhUS for the review. The spelling is correct for most of them, but the case is not for most of them. For the reasons, here is [one](https://workplace.stackexchange.com/q/55384). [Another](https://golang.org/doc/faq#go_or_golang) reason. Another [one](https://stackoverflow.com/q/3989731).

**Pull request merged**